docs/pages/docs/best-practices.mdx

```diff
@@ -70,26 +75,69 @@ Your browser does not support the video tag.
+5. Use periods at the end of a line even in a list.
+6. Use lists aggressively.
+
+The rest are subject to final approval by the developer relations team. In general, it's still a good idea to adopt some of these when writing your next article:
+
+1. Consistent header casing (`I am an example header`, `I Am another Example header`, `I am a Third header`).
+2. Consistent acronyms and concept keyword use (`2-factor`, `two-factor`, `2fa`, `tfa`).
+3. We should avoid using scare-quotes `""` to refer to a sub-service/proper noun - these often have negative connotations associated with them. E.g. - "Trusted Cluster" vs  *Trusted Cluster*. Use italics for concept keywords and tic marks for values. (Clarification: it's harmless to use quotes to approximate or indicate something but not to name or refer to a proper noun. It can be quite conversational and friendly in those other contexts.)
+4. Consistent use of contractions - many teams tend to favor an all or nothing approach here. Contractions keep the language sparse, concise, and conversational. For more formal language, we'd probably want to avoid them.
+5. Use numbered lists for any sequence of steps.
+6. Product proper nouns should be capitalized. `trusted cluster` -> `Trusted Cluster`
+7. Product proper nouns should be bolded. `Trusted Cluster` -> **Trusted Cluster**
+8. Acronyms should always be introduced following a concept keyword and then consistently used thereafter (no concept keyword used) in an article (saves space, typing, and reading - gaurantees the shorthanded term is available).
+
+**Pros:**
+
+- These are relatively minor improvements overall but help to set more uniform documentation standards.
+- Stylistically, this helps to ensure more more professional, neat, and tidy docs.
+- Helps set clear, baseline, guidelines we can expand as documentation needs expand. We can apply these uniformly for blogs, docs, etc.
+- May assist with any future localization efforts since setting a config file for a language is easier when words are one-to-one.
+- Assists with improving searches by content or keyword.
+- Acronyms are occasionally (but only very rarely) associated with multiple concepts. This reduces some potential ambiguities. (Especially for concepts that exist in a complex and particularly acronym-prone space.)
+- These are often helpful for new hires since they can simultaneously be reading and learning the material while performing basic editing.
+- Consistent and helpful styling has an impact on customer satisfaction. Small stylistic changes can improve the efficacy of docs. It's a lot easier to find that *one, crucial, config value* if it's bolded or highlighted.
 
-### Varibles
+## Testing
 
-To prevent updating links and code examples after each release, we can replace them with
-variables. This way, it will be enough to update them in one place.
+1. Make sure to manually test commands and steps!
```

Comment:
Style nit, so is subjective. I think this message is already strong without `!`. But if you feel if it's necessary, you can keep it.